code/_core/obj/item/disk/_disk.dm

```diff
@@ -64,6 +64,61 @@
 		"cost" = 1000
 	)
 
+/obj/item/disk/rcd/conveyor
+	name = "rcd data disk (conveyor belt)"
+	desc = "GET THAT FUKKEN D-- Oh, nevermind."
+	desc_extended = "Stores blueprints for use in an RCD. This one contains schematics for a Conveyor Belt."
+	data = list(
+		"name" = "conveyor_belt.obj",
+		"effect" = /obj/effect/temp/sparkles,
+		"object" = /obj/structure/interactive/conveyor,
+		"cost" = 500
+	)
+
+/obj/item/disk/rcd/diverter
+	name = "rcd data disk (air diverter)"
+	desc = "GET THAT FUKKEN D-- Oh, nevermind."
+	desc_extended = "Stores blueprints for use in an RCD. This one contains schematics for a Air Diverter."
+	data = list(
+		"name" = "air_diverter.obj",
+		"effect" = /obj/effect/temp/sparkles,
+		"object" = /obj/structure/interactive/diverter,
+		"cost" = 500
+	)
+
+/obj/item/disk/rcd/orestorage
+	name = "rcd data disk (ore storage)"
+	desc = "GET THAT FUKKEN D-- Oh, nevermind."
+	desc_extended = "Stores blueprints for use in an RCD. This one contains schematics for a Air Diverter."
+	data = list(
+		"name" = "ore_storage.obj",
+		"effect" = /obj/effect/temp/sparkles,
+		"object" = /obj/structure/interactive/ore_storage,
+		"cost" = 2000
+	)
+
+/obj/item/disk/rcd/oresmelter
+	name = "rcd data disk (smelter)"
+	desc = "GET THAT FUKKEN D-- Oh, nevermind."
+	desc_extended = "Stores blueprints for use in an RCD. This one contains schematics for a Air Diverter."
+	data = list(
+		"name" = "smelter.obj",
+		"effect" = /obj/effect/temp/sparkles,
+		"object" = /obj/structure/interactive/smelter,
+		"cost" = 2000
+	)
+
+/obj/item/disk/rcd/stacker
+	name = "rcd data disk (stacker)"
+	desc = "GET THAT FUKKEN D-- Oh, nevermind."
+	desc_extended = "Stores blueprints for use in an RCD. This one contains schematics for a Air Diverter."
```

Comment:
This also isn't air diverter

Reply:
Fool you've found out my secret! CTRL+C AND CTRL+V!